BUILDING.md

```diff
@@ -0,0 +1,79 @@
+# Building JanusGraph client for Python
+
+-   Building Library:
+
+    A set of scripts have been created to make the job of building the library easier for user, but the scripts
+    expects few pre requisites to be installed before hand. 
+
+-   Pre-Requisites:
+
+    -   Python needs to be installed on system. The library is currently tested against Python
+        3.4, 3.5 & 3.6 only.
+
+    -   Python is added to system path. Make sure, the user can run the following commands from command line:
+        -   python3/ python3.4/ python3.5/ python3.6 on Unix machines.
+        -   python on Windows machines.
+
+    -   Virtualenv needs to be installed. See [PyPi Page](https://pypi.org/project/virtualenv/) for installation steps.
+
+    -   The automated build scripts are tested on following Operating Systems:
+        -   Windows 7
+        -   CentOS
+        -   RHEL 7
+        -   Ubuntu 18.04
+        -   MacOS
+
+-   Building on Windows 7:
+
+    -   A script build.cmd is created which accepts following keyword arguments:
+        -   `d`: Specifies whether documents to be build. Set to true or false.
+        -   `b`: Specifies whether library to be build. Set to true or false.
+
+    -   If any of parameters are missing, the script exits with error message. 
+        An appropriate way to invoke the script would be:
+        ```bash
+        build.cmd "d=true" "b=true"
+        ```
+
+    -   The above example builds the docs, library and also installs to local global Python site-packages.
+
+-   Building on Unix System:
+
+    -   A script named build.sh is created which accepts following parameters:
+
+        -   `-d`: Default to true. Specifies if documentation is to be build.
+        
+        -   `-b`: Defaults to true. Specifies if library is to be build.
+        
+        -   `-p`: The path to Python interpreter on system. If not provided, the script expects python3 is installed\\
+             and the command python3 works on shell and is used to generate virtualenv. If provided, the provided path is \\
```

Comment:
Why the `\\` here?

Reply:
Oh, those are multi line continuation I guess. @mbrukman mentioned in earlier PR comments that the lines needs to be wrapped in around 150 chars per line or something. So the `\\` represents continuition into next line (Like we do in any programming language). Or do you think its not necessary @FlorianHockmann for .MD files?

Reply:
You don't need line continuation markers in `*.md` files for text; just make sure they're indented at the same level (as you've already done), and it will be rendered correctly.

Note that GitHub gives you a preview for all `*.md` files automatically, so you can see how it would render from the code review screen, before the PR is merged.

Reply:
Thanks @mbrukman for quick response. Didn't know about that. Have removed the line continuation :-)

Bdw, again, I don't think I have access to Code review screen only reviewers like you guys can see it right?

Reply:
The info should be public. You can find an analysis of this PR [here](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4938086).

Reply:
You have the same access to the code review screen as any reviewer; just visit your PR (https://github.com/JanusGraph/janusgraph-python/pull/39) and then click on "Files changed" tab at the top which takes you to https://github.com/JanusGraph/janusgraph-python/pull/39/files and then for any `*.md` file, there should be two buttons:

* one that looks like `< >` which is a Markdown (text) diff
* one that looks like a page which is a rich (rendered HTML) diff

See https://help.github.com/en/github/managing-files-in-a-repository/rendering-differences-in-prose-documents for more details.